Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -7864,9 +7862,8 @@ ErrorOr<OwnPtr<CalculationNode>> Parser::parse_a_calculation(Vector<ComponentVal
             continue;
         if (value.is(Token::Type::Delim)) {
             if (first_is_one_of(value.token().delim(), static_cast<u32>('+'), static_cast<u32>('-'), static_cast<u32>('*'), static_cast<u32>('/'))) {
-                // NOTE: Sequential operators are invalid syntax.
                 if (!values.is_empty() && values.last().has<Operator>())
-                    return nullptr;
+                    return Error::from_string_view("Sequential operators are invalid"sv);
```

Comment:
sounds like a jargon… is this a spec error message? if not, then maybe it could be slightly reworked?

(something about juxtaposition or consecutiveness?)